__tests__/app.test.js

```diff
@@ -91,4 +92,32 @@ describe("GET /api/articles/:article_id", () => {
           expect(body.msg).toBe("Bad request");
         });
     });
-  })
\ No newline at end of file
+  })
+  describe("GET /api/articles", () => {
+    test("200: Responds with an array of article objects sorted by date in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+  
+          expect(articles).toHaveLength(articles.length); 
+  
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String),
+              })
+            );
+          });
+          expect(articles).toBeSortedBy("created_at", { descending: true });
```

Comment:
well done for having this assertion - as its testing a different behaviour, we could put it into its own test